SpatialGDK/Documentation/index.md

```diff
@@ -1,58 +1,58 @@
-
 <img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" float: right; margin: 0; display: block; width: 60%; padding: 20px 20px"/>
 
 SpatialOS provides:<br/>
 
 * **Global hosting**: Scalable dedicated hosting for your game in every major gaming region.<br/>
 * **Easy playtesting**: Deploy and test your game from the start of development, and distribute it to your team and players quickly and easily with a ready-made link. Scale-test your build by connecting simulated players.<br/>
 * **Profiling and debugging tools**: Logs and metrics out of the box to help you quickly understand any bugs and performance issues.
-* **Single and multiserver networking**: Use one instance of server software or multiple instances of server software to compute your game world. Multiple servers enable a greater numbers of Actors, players and gameplay systems in your game.</br>
-Multiserver is available through either:
-    * server **offloading** (available in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world, or 
-    * server **zoning** (available in pre-alpha only), in which the game world is split into several geographical areas and each area has a dedicated Unreal server computing all the functionality for it.
+* **Single and multiserver networking**: Use one instance of server software or multiple instances of server software to compute your game world. Multiple servers enable a greater number of Actors, players and gameplay systems in your game.</br>
+
 
-## Get started
 
-If you’re an Unreal game developer and you’re ready to try out the GDK, follow the [Get started]({{urlRoot}}/content/get-started/introduction) guide, or you can [find out more](#find-out-more) before diving in.
+## Multiserver networking
 
-The _Get started_ guide takes you through setting up the GDK and getting the Starter Template project or the Example 
-Project running in the cloud, as well as running locally on your computer. The Example Project gives an overview of the GDK and using SpatialOS, and is the basis for tutorials, and you can use the Starter Template as a basis for your own projects.
+Multiserver functionality is available through either:
 
-After you set up the SpatialOS GDK and the Example 
-Project, you can learn more about the GDK’s functionality with tutorials and guides:
-<img src="{{assetRoot}}assets/example-project/example-project-headline.png" style=" float: right; margin: 0; display: block; width: 60%; padding: 20px 20px"/>
+* server **offloading** (available in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world, or 
+* server **zoning** (available in pre-alpha only), in which the game world is split into several geographical areas and each area has a dedicated Unreal server computing all the functionality for it.
```

Comment:
My text, I think but we probs don't need "only", just reading it again. :-) (While you are here!)
```suggestion
* server **zoning** (available in pre-alpha), in which the game world is split into several geographical areas and each area has a dedicated Unreal server computing all the functionality for it.
```